libs/ledger-live-common/src/device/use-cases/execWithTransport.ts

```diff
@@ -0,0 +1,23 @@
+import Transport from "@ledgerhq/hw-transport";
+import { Exec } from "../../apps";
+import installApp from "../../hw/installApp";
+import installAppWithRestore from "../../hw/installAppWithRestore";
+import uninstallApp from "../../hw/uninstallApp";
+import uninstallAppWithBackup from "../../hw/uninstallAppWithBackup";
+
+export const execWithTransport =
```

Comment:
[COULD] Maybe it's worth adding a bit of TSDoc to explain what this does, and even potentially rename it to something less mysterious...